packages/ui/cypress/support/page-objects/multisigPage.ts

```diff
@@ -13,6 +13,7 @@ export const multisigPage = {
   multisigDetailsContainer: () => cy.get('[data-cy=container-multisig-details]'),
   multisigAccountSummary: () => cy.get('[data-cy=container-multisig-account-summary]'),
   thresholdListItem: () => cy.get('[data-cy=list-item-threshold]'),
+  proxyTypeListItem: () => cy.get('[data-cy=list-item-proxy-type]]'),
```

Comment:
This is not used so far. It'd be great to have a test for it. And verify that the proxy type is correctly displayed.
Maybe in https://github.com/ChainSafe/Multix/blob/79cf6240aaee7926ed4b28d5fd89e17838b9c27a/packages/ui/cypress/tests/watched-accounts.cy.ts ?

Reply:
Yeah it would be good to have a test for this. 

If @bolajahmad is not too familiar with cypress I can add one retroactively let's just make sure to open an issue if that's the case.

Reply:
Oh yes, please open an issue for it, there are also no tests for some related methods, like `thresholdListItem`. I will update this pull request soon.

Reply:
I created the issue, it shouldn't be a hard one https://github.com/ChainSafe/Multix/issues/494